bigtable/client/client_options_test.cc

```diff
@@ -46,3 +46,113 @@ TEST(ClientOptionsTest, EditCredentials) {
   EXPECT_EQ(typeid(grpc::InsecureChannelCredentials()),
             typeid(client_options_object.credentials()));
 }
+
+TEST(ClientOptionsTest, SetGrpclbFallbackTimeout) {
+  bigtable::ClientOptions client_options_object = bigtable::ClientOptions();
+  client_options_object.SetGrpclbFallbackTimeout(5);
+  grpc::ChannelArguments c_args = client_options_object.channel_arguments();
+  grpc_channel_args test_args = c_args.c_channel_args();
+  EXPECT_EQ(2UL, test_args.num_args);
```

Comment:
This and every other `EXPECT_EQ` which checks number of arguments, before you dereference it (e.g., with `test_args.args[1]`) must be `ASSERT_EQ` because if this fails, the test keeps going, but the binary will segfault.

Any check, past which it doesn't make sense to go on (or especially, it's unsafe to keep going) should be an `ASSERT`. Anything which is just a check that doesn't impact downstream code should be an `EXPECT` so we can get as much information (failure messages) from a single test run as possible.

Reply:
Done.